examples/example.php

```diff
@@ -64,7 +78,7 @@
 //    $line2->setVat('06');
 //    $sale->addLine($line2);
 //
-    $response = $teamleader->opportunitiesAddSale($sale);
+//    $response = $teamleader->opportunitiesAddSale($sale);
```

Comment:
I don't know the code, so I can't really say if it's wrong or not. It just did seem strange to me that you commented one extra line of code, without replacing something. But if this is just some example, it doesn't really matter.